iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/static_storage.hpp

```diff
@@ -39,6 +39,9 @@ namespace cxx
 ///       optimized_storage would have a dynamic memory fallback when static memory is
 ///       insufficent.
 template <uint64_t Capacity, uint64_t Align = 1>
+/// @NOLINTJUSTIFICATION static_storage provides uninitialized memory, correct initialization is the users
```

Comment:
Same as mentioned [here](https://github.com/eclipse-iceoryx/iceoryx/pull/1495#discussion_r920005056), the `@` can be removed.

```suggestion
/// NOLINTJUSTIFICATION static_storage provides uninitialized memory, correct initialization is the users
```